### PR TITLE
Adds transition:name and transition:animate attributes

### PR DESCRIPTION
**Pull request by matthewp**
## Changes

- Specifies `transition:name` and `transition:animate`

## Testing

I did not :( Should I have?

## Docs

N/A

### CODE REVIEW

**Review by ematipico — APPROVED** [review]

File: .changeset/witty-yaks-lick.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/language-server': patch
```
Comment (Princesseuh) [inline .changeset/witty-yaks-lick.md:2]: needs to be a patch for the VS Code extension as well

File: packages/language-server/src/plugins/html-data.ts [hunk 2]
```diff
@@ -180,6 +180,26 @@ export const astroAttributes = newHTMLDataProvider('astro-attributes', {
...
+			references: [
+				{
+					name: 'Astro reference',
+					url: 'https://docs.astro.build/en/guides/view-transitions/#transition-directives',
+				},
+			],
+		},
+		{
+			name: 'transition:name',
+			description: 'Species a `view-transition-name` for this element. The name should be unique on the page',
```
Comment (Princesseuh) [inline packages/language-server/src/plugins/html-data.ts:195]:
```suggestion
			description: 'Specifies a `view-transition-name` for this element. The name should be unique on the page.',
```

File: packages/language-server/src/plugins/html-data.ts [hunk 3]
```diff
@@ -180,6 +180,26 @@ export const astroAttributes = newHTMLDataProvider('astro-attributes', {
 				},
 			],
 		},
+		{
+			name: 'transition:animate',
+			description: 'Specifies an animation to use with this element on page transition',
```
Comment (Princesseuh) [inline packages/language-server/src/plugins/html-data.ts:185]:
```suggestion
			description: 'Specifies an animation to use with this element on page transition.',
```

Comment (matthewp) [reply packages/language-server/src/plugins/html-data.ts:185]: None of the other descriptions have periods. Sure you want one here?

**Review by Princesseuh — APPROVED** [review]

**Pull request merged**